pyxem/generators/integration_generator.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this supposed to be modified here? isn't this exactly the same as the master?